source/_integrations/olarm.markdown

```
@@ -0,0 +1,127 @@
+
+{% include integrations/config_flow.md %}
+
+## Entities
+
+The **Olarm** {% term integration %} provides the following {% term entities %}.
+
+### Binary sensors
+
+The integration creates various binary sensors depending on your alarm system configuration:
+
+#### Zone sensors
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)